packages/dev/core/src/ShadersWGSL/ShadersInclude/bumpVertex.fx

```diff
@@ -3,6 +3,9 @@
 		var tbnNormal: vec3f = normalize(normalUpdated);
 		var tbnTangent: vec3f = normalize(tangentUpdated.xyz);
 		var tbnBitangent: vec3f = cross(tbnNormal, tbnTangent) * tangentUpdated.w;
-		vTBN =  mat3x3f(finalWorld) *  mat3x3f(tbnTangent, tbnBitangent, tbnNormal);
+		var matTemp =  mat3x3f(finalWorld[0].xyz, finalWorld[1].xyz, finalWorld[2].xyz) *  mat3x3f(tbnTangent, tbnBitangent, tbnNormal);

```

Comment:
I think you can use `let` instead of `var` (and in the lines above too). I don't know if using `let` will lead to better perf, but it's possible (`let` in wgsl == `const`in js, basically).